plugins/Invoke-IcingaCheckHTTPStatus.psm1

```diff
@@ -66,6 +66,8 @@
 .PARAMETER ConnectionErrAsCrit
     By default the plugin will return UNKNOWN in case a connection to a webserver is not possible. By using this
     flag, the result will be modified from UNKNOWN to CRITICAL
+.PARAMETER IgnoreSSL
+    Use this flag to ignore SSL errors in case your endpoints are not trusted by the client or you are using self-signed certificates.
```

Comment:
The name `IgnoreSSL` is somewhat misleading, since it suggests (in my mind) that TLS itself will be ignored (deactivated?), not only a failure to verify a certificate chain.
`IgnoreTLSCertificateValidation` might be quite a mouthful, but it might express better what the option will (and should) do

Reply:
The issue here is, that the HTTP Json plugin already uses the term "IgnoreSSL". Therefore this would match the other plugin and based on the description, it should be pretty obvious that when you are using an HTTP plugin the "IgnoreSSL" flag will just ignore TLS errors, shouldn't it?

Reply:
As already discussed, offline, I do get your point, but the other Web plugin we provide uses the same flag. It would make things weird for now if they are not alligned.

We should change both values later on if there is a real need to change it, as discussed, in my point of view, SSL might be outdated, the provided error message from Windows will contain both.